samples/contrib/versioned-pipeline-ci-samples/helloworld-ci-sample/README.md

```diff
@@ -0,0 +1,17 @@
+# Hello World CI Sample
+
+## Overview
+
+This sample uses cloudbuild to implement the continuous integration process of a simple pipeline that outputs "hello world" to the console. Once all set up, you can push your code to github repo, then the build process in cloud build will be triggered automatically, then a run will be created in kubeflow pipeline. You can view your pipeline and the run in kubeflow pipelines. 
```

Comment:
Maybe use "Cloud Build" instead of cloud build? Sort of a brand name?

Reply:
Same for "Kubeflow Pipelines"

Reply:
ok, will do.

Reply:
gentle bump